src/ui.rs

```diff
@@ -6,7 +6,8 @@ use ratatui::{
     layout::{Alignment, Constraint, Direction, Layout},
     style::{Modifier, Style},
     text::{Span, Text},
-    widgets::{Block, BorderType, Borders, Cell, List, ListItem, ListState, Row, Table, Paragraph},
+    prelude::Rect,
```

Comment:
`prelude::Rect` → `layout::Rect`

Personally I don't like the prelude imports (they are meant to be used as `prelude::*`) but other ratatui people seem to want them.